Week 04/Lecture 08/Assignment 3/src/main/java/com/lecture8/assignment3/config/AppConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.lecture8.assignment3.config;
+
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppConfig {
+
+    
+    @Bean
+    @Qualifier("dataSource1")
```

Comment:
Should separete datasource bean into another class (only for DataSource config)

Reply:
DataSource configurations have been moved to a new class named DataSourceConfig.java. This class is responsible solely for configuring the data sources.

**Changes Made:**

- Created DataSourceConfig.java for data source configurations.
- Removed data source configuration from AppConfig.java.